src/lib/execPromise.ts

```diff
@@ -73,13 +84,11 @@ export const execPromise = ({
 
         childProcess.kill();
 
-        rej(
-          new Error(
-            `Execution timed out: command didn't yield any result in ${color.yellow(
-              `${timeout}ms`
-            )}\n${debugInfo}`
-          )
-        );
+        res({
+          errorMessage: `${getCommandFailedMessage()}. Reason: Execution timed out: command didn't yield any result in ${color.yellow(
```

Comment:
> command didn't yield
what command? or will it be inside `getCommandFailedMessage`?

Reply:
ditto: suggest to move message to a separate variable to make it easier to read

Reply:
The command will be inside getCommandFailedMessage. I can move this to a variable. Not sure if it really impores readability at all